### ISSUE TITLE
New Feature: BLE sensor

### ISSUE THREAD

**Issue by gion86** [mendhak/gpslogger#660]
Hello, is everyone interested in Bluetooth Low Energy support for this app?
So one can bind an heart beat sensor for example and have all the informations (GPS track and heartbeat) in the same file. It could be exported as .tcx file..

Thankyou!

**Comment by vincele** [comment]
Yes, I would be very interested in such support, I may even be able/willing to help.
Is upstream interested in merging this kind of thing ?
Is there an alternative FOSS app that can already do it ?
